src/responder/common/cache_req/plugins/cache_req_autofs_entry_by_name.c

```diff
@@ -0,0 +1,149 @@
+/*
+    Authors:
+        Pavel Březina <pbrezina@redhat.com>
+
+    Copyright (C) 2016 Red Hat
```

Comment:
nitpicking: can you please fix the year? (In other added files too)

Reply:
Fixed.